usr/lib/ep11_stdll/ep11tok.conf

```diff
@@ -53,6 +53,31 @@
 #
 #      USE_PRANDOM
 #
+# To optimize encrypt/decrypt and sign/verify performance, a corresponding
+# protected key can be created for AES secure keys and added to the secure key
+# via IBM specific attribute CKA_IBM_OPAQUE_PKEY. This protected key is then
```

Comment:
Do we really want to mention the CKA_IBM_OPAQUE_PKEY attribute name here, although we do not allow that attribute to bet get or modified from an application?

Reply:
Hmmm ... I tried to explain a bit how all this happens. But yes, as we will document the protected key option in our ep11 book anyway, we may leave this info out here. Done.